rds-deployment-ha-broker/Scripts/RdcbHaPostConfig.ps1

```diff
@@ -20,7 +20,7 @@ param
      [string]$clientAccessName,
      [string]$sqlClientUrl,
      [string]$dnsServer,
-     [string]$sqlServer
+     [string]$sqlServer=""
```

Comment:
are changes to the script here really required?

what is the scenario where sqlServer is null/empty string?